vowpalwabbit/core/src/parse_args.cc

```diff
@@ -883,6 +887,27 @@ void parse_feature_tweaks(options_i& options, VW::workspace& all, bool interacti
     }
   }
 
+  if (options.was_supplied("ignore_features_dsjson_experimental"))
+  {
+    for (const auto& ignored : ignore_features)
+    {
+      std::tuple<std::string, std::string> namespce_and_feature = VW::extract_ignored_feature(ignored);
+      const std::string& ns = std::get<0>(namespce_and_feature);
+      const std::string& feature_name = std::get<1>(namespce_and_feature);
+      if (!(ns.empty() || feature_name.empty()))
```

Comment:
Can we ignore root namespace features? Looks like right now neither "--ignore $feature" nor "--ignore |$feature" will not enter this branch

Reply:
good catch, fixed